.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -1,43 +1,18 @@
 name: Report a bug
 description: Any errors that are not UI related.
-labels: ["needs triage", "bug"]
```

Comment:
why did you remove this?

Reply:
Suggestion from @carmocca here: https://github.com/Lightning-AI/lightning/pull/14193#discussion_r972920380